net/seafile-server/Makefile

```diff
@@ -174,9 +171,6 @@ define Package/seafile-server/install
 		$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$$$$bin $(1)/usr/libexec/ ; \
 	done
 
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/python/seafile/*.py $(1)$(PYTHON_PKG_DIR)/seafile/
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/python/seaserv/*.py $(1)$(PYTHON_PKG_DIR)/seaserv/
```

Comment:
should this be `PYTHON3_PKG_DIR` ?
or do we need it?

Reply:
I added a separate [python3-seafile-server](https://github.com/openwrt/packages/blob/76bd346085ce50c05c27ea1d6fa1452e015f36d7/net/seafile-server/Makefile#L109) package where those files are [placed](https://github.com/openwrt/packages/blob/76bd346085ce50c05c27ea1d6fa1452e015f36d7/net/seafile-server/Makefile#L212-L213) in `PYTHON3_PKG_DIR`.

Reply:
ack